source/user-guide/getting-aws-credentials.html.md.erb

```diff
@@ -13,14 +13,14 @@ You can obtain temporary AWS credentials to use the AWS CLI or other command lin
 
 To have access to your AWS credentials you will need to be a member of the GitHub team specified when the [environment was created](./creating-environments.html#github-team-slug)
 
-Note - you do not need to obtain AWS credentials to deploy infrastructure, this is done via GitHub Workflows (see [deploying you infrastructure](deploying-your-infrastructure.html).
+Note - you do not need to obtain AWS credentials to deploy infrastructure, this is done via GitHub Workflows (see [deploying you infrastructure](deploying-your-infrastructure.html)).
 
 If you need credentials to make an application deployment, a CI user is created as part of the initial account set up, see [here](obtaining-deployment-credentials.html) for obtaining the credentials for that user.
 
 1. [Log in to the AWS Console](accessing-the-aws-console.html)
 
-1. Select the account you wish to access programatically
+2. Select the account you wish to access programmatically
 
-1. Choose the `Command line or programmatic access` options
+3. Choose the `Command line or programmatic access` options
 
-1. Follow the instructions on screen to retrieve your credentials.
+4. Follow the instructions on screen to retrieve your credentials.
```

Comment:
FWIW, markdown will still display this in numerical order even if the source doesn't.